tutorials/getting-started/background.js

```diff
@@ -0,0 +1,6 @@
+let color = '#3aa757';
```

Comment:
- Can this variable be local to the onInstalled callback. 
- Also s/color/defaultColor
- Also can this be const?